app/guid-node/metadata/-components/metadata-select/template.hbs

```diff
@@ -0,0 +1,18 @@
+<div local-class='metadata-select-container {{if this.isMobile 'mobile'}}'>
+    <div local-class='title' data-test-select-title>
+        {{ this.template.schemaName }}
+    </div>
+    <div local-class='description' data-test-select-description>
+        {{ this.template.template.description}}
```

Comment:
Will the `CedarMetadataTemplate.template` object attribute always have a `description`? If they may not, we should wrap this `<div local-class='description'>` in a `{{#if this.template.template.description}}` statement so we don't get an extra 20px of unneeded whitespace